src/basecamp.h

```diff
@@ -116,9 +118,12 @@ class basecamp
         bool start_garage_chop( npc &p, const std::string dir, const tripoint &omt_tgt );
         void start_farm_op( npc &p, const std::string &dir, const tripoint &omt_tgt, farm_ops op );
 
-        // mission return functions
+        // mission functions
+        void camp_missions( mission_data &mission_key, npc &p );
+        bool handle_camp_mission( mission_entry &cur_key, npc &p );
```

Comment:
```suggestion
        void camp_missions( mission_data &mission_key );
        bool handle_camp_mission( mission_entry &cur_key );
```

No NPCs at this point!